src/utils/daos_hdlr.c

```diff
@@ -2448,8 +2449,12 @@ dfuse_count_query(struct cmd_args_s *ap)
 	rc = ioctl(fd, DFUSE_IOCTL_COUNT_QUERY, &query);
 	if (rc < 0) {
 		rc = errno;
-		DH_PERROR_SYS(ap, rc, "ioctl failed");
-		rc = daos_errno2der(errno);
+		if (rc == ENOTTY) {
+			rc = -DER_MISC;
```

Comment:
This doesn't' change the value of rc but it does remove the extra output.  I've filed https://daosio.atlassian.net/browse/DAOS-13977 to remove -DER_MISC across this command.